workers/tests/unittests/test_config.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import os
+import tempfile
+import unittest
+from unittest import mock
+
+from ohsome_quality_analyst.config import config
+
+
+class TestConfig(unittest.TestCase):
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_get_config_path_empty_env(self):
+        self.assertEqual(
+            config.get_config_path(),
+            os.path.join(
+                os.path.dirname(
+                    inspect.getfile(config),
+                ),
+                "config.yaml",
+            ),
+        )
```

Comment:
I would write this test differently and not copy the definition of the method into the unittest. I would compare it explicitly with the path string you are using.

Reply:
The expected path is now build relative to the test file.